frontend/packages/topology/src/components/utils/checkResourceQuota.tsx

```diff
@@ -0,0 +1,37 @@
+import { getUsedPercentage } from '@console/app/src/components/resource-quota/utils';
+import { ResourceQuotaModel } from '@console/internal/models';
+
+export const checkQuotaLimit = (resourceQuotas: any) => {
```

Comment:
will be nice to add unit test for this util and type for return types

Reply:
Added unit test for the util and added return type